lib/usherbot.js

```diff
@@ -108,6 +109,18 @@ u.handleVote = function( movie ){
 	return this.messages.success.vote.replace( "{movie}", movie );
 }
 
+// CHECK TO SEE IF MOVIE IS AVAILABLE TO STREAM
+u.searchAvailableStream = function( movie ) {
+    StreamIt.search(movie)
+    .then(function(results) {
+        //return StreamIt.streaming(results[0]);
+        console.log(results);
```

Comment:
What does this return? (string, array, etc)
We can probably just add a string into our message content file for stream movies like "This movie can be streamed on " and append whatever results onto it.